packages/governance/test/swingsetTests/contractGovernor/governedContract.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { E } from '@agoric/eventual-send';
+import { assert, details as X } from '@agoric/assert';
+import { sameStructure } from '@agoric/same-structure';
+
+import { buildParamManager, ParamType } from '../../../src/paramManager';
+
+const MALLEABLE_NUMBER = 'MalleableNumber';
+
+export const governedParameterTerms = {
+  contractParams: [MALLEABLE_NUMBER],
```

Comment:
Hmm, why this particular structure? It seems to have an unnecessary level. Could it just be:

```
export const governedParameterTerms = [MALLEABLE_NUMBER];
```

Reply:
Why export this?